lib/screens/publish_screen.dart

```diff
@@ -74,9 +73,9 @@ class _PublishScreenState extends State<PublishScreen> {
                     easelProvider.isVideoLoading = true;
                     homeViewModel.previousPage();
                   },
-                  icon: const Icon(
+                  icon: Icon(
                     Icons.arrow_back_ios,
-                    color: EaselAppTheme.kWhite,
+                    color: easelProvider.nft.assetType == kPdfText ? EaselAppTheme.kGrey : EaselAppTheme.kWhite,
```

Comment:
Why this icon is grey?

Reply:
because incase of pdf we can't see the back icon due to same color